spec/models/section_spec.rb

```diff
@@ -4,6 +4,12 @@
 
 	let(:section){ FactoryGirl.build(:section) }
 
+	describe 'can behave like an uid class' do
+    context 'when behaves like uid' do
+      it_behaves_like 'uid'
+    end
+  end
+
```

Comment:
@marcellosachs You can drop the `describe` and `context` here, they add not much in the way of understanding and it will just work if you have `it_behaves_like 'uid'`.